### PR TITLE
Bump version to v6.0.0

### PR DESCRIPTION
**Pull request by DHUKK**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "xocto"
-version = "5.1.0"
+version = "5.1.1"
```
Comment (mmxw) [inline pyproject.toml:7]: should this be 6.0.0?

Comment (DHUKK) [reply pyproject.toml:7]: Sorry yes, made that change before I noticed the breaking change 🤦

Comment (DHUKK) [reply pyproject.toml]: Fixed

**Review by mmxw — APPROVED** [review]
🚀 🙏

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,13 @@
 
 ## Unreleased
 
+
+## v6.0.0 - 2024-08-30
+
 - [Breaking] Use parameter object for passing options to `pact_service`.
   Rename parameter `pact_version` to `consumer_version`.
-  [#167](https://github.com/octoenergy/xocto/pull/167)
+  [#166](https://github.com/octoenergy/xocto/pull/166)
```
Comment (DHUKK) [inline CHANGELOG.md:10]: This PR link was incorrect

**Review by noelia-mari — APPROVED** [review]
LGTM!

**Pull request merged**